<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rascally</class>
 <widget class="QMainWindow" name="Rascally">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/assets/icon_1.ico</normaloff>:/icons/assets/icon_1.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 12, 36);
color: white;
border-radius: 20px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="title_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="titleLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="title">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Rascally</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/assets/cross.png</normaloff>:/icons/assets/cross.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="central_frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="centralLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="engines_frame">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="leftLayout">
          <item>
           <widget class="QLabel" name="engines_label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>38</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
background-color: rgb(85,0,0);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string notr="true">Engines</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="enginesArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QWidget" name="enginesContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>288</width>
               <height>579</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="enginesLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="config_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string notr="true">Configuration</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="Wwelcome">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QFrame" name="welcome_frame">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/images/assets/mapache_1.png);
background-repeat: no-repeat;
background-position: center;
background-attachment: fixed;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Noto Serif Armenian Medium</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: grey;</string>
             </property>
             <property name="text">
              <string>Designed and developed by Franje-Gueje                                                                                                                  https://github.com/FranjeGueje/rascally       </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="WConfiguration">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="steamPathLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Steam Path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="steamPathLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background-color: rgb(22, 0, 22)
}

QLineEdit:focus
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="steamGridDBKeyLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>SteamGridDB key</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="steamGridDBKeyLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background-color: rgb(22, 0, 22)
}

QLineEdit:focus
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lnkLocationLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Lnk Location</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lnkLocationLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
background-color: rgb(22, 0, 22)
}

QLineEdit:focus
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="clonningModeLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Clonning Mode</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="clonningModeComboBox">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>SYMLINK CLONE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BALANCED CLONE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">FULL CLONE</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="discardRepeatedGamesLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string notr="true">Discard Repeated Games</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="discardRepeatedGamesCheckBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="compatdataExceptionsLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Compatdata Exceptions</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QListWidget" name="exceptionList">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QListWidget{
background-color: rgb(22, 0, 22)
}

QListWidget:focus
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="spacing">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="deleteExceptionButton">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
             </property>
             <property name="text">
              <string>Delete Exception</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="logLevelLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Log Level</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="logLevelComboBox">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string notr="true">Critical</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Error</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Warning</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Info</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Debug</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="8" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="saveConfigButton">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
    border-width: 1px;
    border: 1px solid white;
    border-color: white;
	border-radius: 10px;
}</string>
             </property>
             <property name="text">
              <string>Save Configuration</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Wgames">
          <layout class="QVBoxLayout" name="rightLayout">
           <item>
            <widget class="QScrollArea" name="gameArea">
             <property name="cursor" stdset="0">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="gamesContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>88</width>
                <height>18</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gamesLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar{
    border: 2px solid rgb(36, 12, 36);
	border-radius: 5px;
}

QProgressBar::chunk{
	background-color: rgb(85,0,0);
    border-radius: 5px;
}
</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
